### PR TITLE
support image import for Alicloud

### PR DESCRIPTION
**Pull request by jia-jerry**
**What this PR does / why we need it**:
Implements scripts to import image from S3 for Alicloud
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: ci/glci/alicloud.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+from aliyunsdkecs.request.v20140526 import DescribeRegionsRequest
+from aliyunsdkecs.request.v20140526 import CopyImageRequest
+from aliyunsdkcore.client import AcsClient
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+TIME_OUT = 30 * 60  # in seconds, 30 Minutes
+
+
+class AlicloudImageStatus:
```
Comment (ccwienk) [inline ci/glci/alicloud.py:22]: nit: rather use `enum.Enum` as base class (this should be a one-line-change) -->
```
class AlicloudImageStatus(enum.Enum):
...
```

File: ci/glci/model.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 import yaml
 
 own_dir = os.path.abspath(os.path.dirname(__file__))
-repo_root = os.path.abspath(os.path.join(own_dir, os.path.pardir, os.path.pardir))
+repo_root = os.path.abspath(os.path.join(
+    own_dir, os.path.pardir, os.path.pardir))
```
Comment (ccwienk) [inline ci/glci/model.py:15]: nit: place closing paranthesis in new line? this is not an "official" recommendation as of PEP8, but would be consistent with the other code from `ci`

File: ci/glci/model.py [hunk 3]
```diff
@@ -25,8 +26,8 @@ class FeatureType(enum.Enum):
     MODIFIER = 'modifier'
 
 
-Platform = str # see `features/*/info.yaml` / platforms() for allowed values
-Modifier = str # see `features/*/info.yaml` / modifiers() for allowed values
+Platform = str  # see `features/*/info.yaml` / platforms() for allowed values
```
Comment (ccwienk) [inline ci/glci/model.py:29]: nit: revert whitespace-only-diff

File: ci/promote.py [hunk 4]
```diff
@@ -35,7 +35,7 @@
     },
     'root': {
         'level': logging.DEBUG,
-        'handlers': ['console',],
+        'handlers': ['console', ],
```
Comment (ccwienk) [inline ci/promote.py:38]: I suppose this is from auto-code-formatter?

File: ci/requirements.txt [hunk 5]
```diff
@@ -7,3 +7,6 @@ gardener-cicd-base
 google-api-python-client
 google-auth
 google-cloud-storage
+aliyun-python-sdk-ecs==4.19.7
```
Comment (ccwienk) [inline ci/requirements.txt:10]: please sort alphabetically

File: ci/promote.py (hunk 4 above)
Comment (jia-jerry) [reply ci/promote.py:38]: yes... autopep8

**Pull request merged**